uui/components/pickers/PickerInput.tsx

```diff
@@ -62,6 +62,10 @@ export class PickerInput<TItem, TId> extends PickerInputBase<TItem, TId, PickerI
         ) : (
             <DataPickerRow
                 { ...rowProps }
+                rawProps={ {
+                    ...rowProps.rawProps,
+                    'data-testid': `uui-PickerInput-item-${rowProps.rowKey}`,
```

Comment:
as far as I remember we agreed to use semantic markup for tags, i.e. add aria-role to the wrapper div and to each Tag item. Adding "data-testid" is less preferrable.